rfcs/formats/identity.md

```diff
@@ -0,0 +1,203 @@
+
+### Multi-signature based verificationMethod
+
+A network DID must be controlled jointly by the network's participant units for operations such as creation of the DID, and its updation with changing network structure, and for other operations such as rotating keys, updating verification methods etc.. To enable this group control we introduce a verificationMethod that uses multi-signatures and policies.
+
+The verificationMethod will have the following properties:
+
+**id** - This will be the Network DID followed by a fragment forming a DID URI that points to this verification method. Eg: `did:<iin_name>:<network_name>#unique_verificationMethod_name`
+
+**type** - `BlockchainNetworkMultiSig`
+
+**controller** - Controller of this DID must be the network itself, and thus its value should be same as the Network DID.
+
+**Verification Materials for BlockchainNetworkMultiSig:**
+
+**multisigKeys** - Map of participant DID to its verificationMethod. This contains one entry for each participant unit in `networkParticipants` list. The key is the participant unit's DID and the value is a DID URI to the verification method of the participant DID.
+
+
+Eg. If `network_participant_1` has a DID `did:<iin_name>:<network_participant_1>`, then `multisigKeys` will contain one verification method of the DID Document like `"did:<iin_name>:<network_participant_1>":"did:<iin_name>:<network_participant_1>#verificationMethod1"`
+
+
+For creation of Network DID, the IIN registry must authenticate the DID creation request based on the `BlockchainNetworkMultiSig` verification method. This implies that the request must be authenticated based on each of the verificationMethods specified in  `multisigKeys`.
+
+
+> NOTE on extensibility: The `networkParticipants`  lets any DID to be specified, with a verification mehtod in `BlockchainNetworkMultiSig.multisigKeys`.  Thus another Network DID can also be represented as a participant unit of a network. This can be very useful in certain use cases where different blockchain networks may be stakeholders of a bigger network.
+
+
+
+**updatePolicy** - Defines a combination of participant units that must sign and authenticate a Network DID updation request. This `updatePolicy` has the condition format as specified in [`verifiablecondition2021`](https://w3c.github.io/did-spec-registries/#verifiablecondition2021) verification method. It can be used to combine different verification methods together to form conjugated conditions such as logical operations, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
+
+When a blockchain network changes with participant members leaving and new participants joining, the existing members of the network can update the Network DID as long as they can authenticate the update request by collecting enough signatures to satisfy the `updatePolicy`.
```

Comment:
Just wondering, if we have a policy that requires a unanimous vote (i.e., signatures from all participants) to update a network DID, and then one or more of the members leave, just the votes of the remaining members won't satisfy the network update policy, right? What do we do in this situation? (We don't face a similar situation when new members join.)

Reply:
Will create a separate page on update policies.

Reply:
Yea, such strict policies could in theory cause an issue and would have to be a consideration when crafting update policies. In practice though, in an enterprise context, I would imagine there would be structured off-boarding processes when an organization leaves a network which could involve it also voting on an update that removes it from the member list. Of course, this might not be the case in less planned scenarios in which an organization effectively exits a network (e.g. company going bankrupt, sudden sanctions/embargos...)

Reply:
I have added a separate file for the Network DID update policies:
https://github.com/ghoshbishakh/weaver-dlt-interoperability/blob/rfc-identity/rfcs/models/identity/identity-update-policy.md